### DISCUSSION TITLE
Extra fonts not appear

### DISCUSSION THREAD

**Discussion by hys0star** [nextcloud/all-in-one#4726] [Questions] [answered]
I uploaded some Extra fonts in Nextcloud settings but they don't appear at all. What could be the problem?

![Screenshot 2024-05-29 at 02-47-43 Office - Administration settings - 황윤성 클라우드](https://github.com/nextcloud/all-in-one/assets/87403078/57426dcd-4d27-4143-b630-052294d49c67)
![스크린샷 2024-05-29 024738](https://github.com/nextcloud/all-in-one/assets/87403078/8f6db1c1-c78a-4594-a3d1-eb7b96599d3a)

**Comment by szaimen** [comment, marked as answer]
Hi, I think this is a bug in Collabora. See e.g. https://github.com/CollaboraOnline/online/issues?q=is%3Aissue+is%3Aopen+%22remote+font%22+
